scripts/bb-dependencies.sh

```diff
@@ -45,6 +45,11 @@ CentOS*)
         zlib-devel libuuid-devel libblkid-devel libselinux-devel \
         xfsprogs-devel libattr-devel libacl-devel libudev-devel \
         device-mapper-devel openssl-devel
+
+    # Packages that are version dependent and not always available
+    if cat /etc/centos-release | grep -Fq 7.; then
```

Comment:
Is this check sufficient? Did CentOS 6 have patched version that would create a version number similar to `6.7.123`? If so, this check would succeed when it should fail on CentOS 6.7.

Reply:
CentOS does do that for 7.x, but not 6.x so it works out. Here's a small selection of version strings on systems I have access to:
<pre>
CentOS release 6.9 (Final)
Red Hat Enterprise Linux Server release 6.5 (Santiago)
CentOS Linux release 7.4.1708 (Core) 
</pre>
But since I'm targeting 7.x specifically this is fine.

Reply:
Awesome, thanks! Just wanted to be sure.